README.md

```diff
@@ -516,6 +517,25 @@ as a `debug-each` script at the project, backend, and suite levels, so they
 are aggregated and repeated for every task under them.
 
 
+<a name="reexecuting"/>
+
+## Reexecuting
+
+Reproduce an error could be a very boring experience, and Spread has a way to
+simplify that process by reexecuting the tasks as many times is desired until
+the task fails.
+
+To do that there are two options `-retries`, and `-stop-on-fail` which are
```

Comment:
Retrying seems to have the meaning of _trying again if it fails_ but what we mean here is _run again until it fails_, so something like _-repeat 5_ would be more clear.

I also suggest getting rid of _-stop-on-fail_ and making that the default behavior. We don't want to watch the same task break 5 different times, perhaps in different ways. Every single failure is important and should be addressed.